homework/Vasili_Zasinets/NumberTask/src/main/java/com/tutrit/java/quickstart/homework/number/NumberService.java

```diff
@@ -0,0 +1,75 @@
+package com.tutrit.java.quickstart.homework.number;
+
+public class NumberService {
+   private int result;
+   private int i;
```

Comment:
There is no need do use this vars as service properties, because they need only to store intermediate state.

Reply:
ok